public/CSS Files/style.css

```diff
@@ -0,0 +1,215 @@
+@import url('https://fonts.googleapis.com/css?family=Oxanium|Trade+Winds&display=swap');
+
+.grid-container {
+    display: grid;
+    width: 850px;
+    height: 700px;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 2fr 8fr 2fr;
+    grid-gap: 1rem;
+    grid-template-areas:
+        "header  map map"
+        "sideBar  map  map"
+        "footer  footer    footer";
```

Comment:
The grid looks to be working as it should, but in your ```.header``` rule on line 28, I believe you want to also add the rule ```grid-area: header``` just to be safe, and in case you wanted to rearrange the way your grid was set up, then the .header element would know where to place itself.